connector/src/test/scala/com/microsoft/kusto/spark/KustoWriterTests.scala

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.kusto.spark
+
+import java.io.BufferedWriter
+import java.nio.charset.StandardCharsets
+import java.util.zip.GZIPOutputStream
+
+import com.microsoft.kusto.spark.datasink.{FileWriteResource, KustoWriter}
+import com.univocity.parsers.csv.{CsvWriter, CsvWriterSettings}
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.catalyst.InternalRow
+import org.mockito.Mockito._
+import org.scalatest.{FunSpec, Matchers}
+
+
+class KustoWriterTests extends FunSpec with Matchers {
```

Comment:
We used FlatSpec in all the other tests. It would be better to have all tests aligned in style.